src/analysis/normed_space/basic.lean

```diff
@@ -80,6 +92,10 @@ instance semi_normed_comm_ring.to_comm_ring [β : semi_normed_comm_ring α] : co
 @[priority 100] -- see Note [lower instance priority]
 instance normed_ring.to_normed_group [β : normed_ring α] : normed_group α := { ..β }
 
+@[priority 100] -- see Note [lower instance priority]
+instance normed_division_algebra.to_normed_group [β : normed_division_ring α] : normed_group α :=
```

Comment:
I don't think this instance should exist - instead you should have
```suggestion
instance normed_division_ring.to_normed_ring [β : normed_division_ring α] : normed_ring α :=
```

Reply:
I already had the `to_normed_ring` one, I'm not sure why I add this. Removed.